components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -77,14 +81,21 @@ class ExperimentsTest {
         `when`(experimentsUpdater.getExperimentSource(configuration)).thenReturn(experimentSource)
     }
 
+    private fun mockUpdaterWorkManagerFunctionality() {
```

Comment:
I was nervous about running everything async, but I like the idea of adding _limited_ async testing since it feels like it would be easier to manage/control a test or two running async rather than everything.